packages/api/schema/stryker-core.json

```diff
@@ -349,6 +361,10 @@
       "description": "The options for the html reporter",
       "$ref": "#/definitions/htmlReporterOptions"
     },
+    "jsonReporter": {
+      "description": "The options for the json reporter",
+      "$ref": "#/definitions/jsonReporterOptions"
```

Comment:
Please fill the `default` here with `{}`. That way, you don't need to check if `options.`jsonReporter` is null later, or duplicate the default value of "jsonReporterOptions.baseDir".